beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -52,20 +74,77 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
     );
   };
 
+  const setMoveHighlight = () => {
+    setTerritoryFill(theme.palette[userCountry].main);
+    setTerritoryFillOpacity(0.9);
+    setTerritoryStrokeOpacity(1);
+  };
+
+  const setCapturedHighlight = (country) => {
+    if (country) {
+      if (country === "none") {
+        setTerritoryFill("none");
+      } else {
+        setTerritoryFill(theme.palette[country].main);
+      }
```

Comment:
lets do a ternary for this check.It would get rid of 1 if else statement.
`country === "none" ? setTerritoryFill("none") : setTerritoryFill(theme.palette[country].main)`